events/dbpersist.go

```diff
@@ -28,10 +28,11 @@ type RepoEventRecord struct {
 	Commit util.DbCID
 	Prev   *util.DbCID
 
-	Time  time.Time
-	Blobs []byte
-	Repo  util.Uid
-	Event string
+	Time   time.Time
+	Blobs  []byte
+	Repo   util.Uid
+	Event  string
+	Rebase bool
```

Comment:
Good questions:
- `TooBig` is handled elsewhere (inside of the hydration method) since its definition is simply depenent on the hydrated structure, I didnt see much reason to put it in the DB (though, thinking about it now, it probably couldnt hurt and might save us from an extra disk access)
- Since storing arrays of things in sql is complicated, `Blobs` is just the cids array marshaled to json. Could obviously do this better in the future, but I think once we get to the point of that being a problem we need to move to a more specialized datastore for event persistence (fancy nvme things)
- We pull blocks out of the carstore on demand to avoid storing everything multiple times. This might change with a different persistence engine, we might decide to just write the fully hydrated event to disk, but for now i'm sparing postgres from having a bajillion toast files